From cffc0bffe9cf88e04a6008004f9cc33a72cd568e Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:37:42 -0400
Subject: [misc] mei: drop pr_fmt macros

Message-id: <1399041470-21330-70-git-send-email-prarit@redhat.com>
Patchwork-id: 79361
O-Subject: [RHEL7.1 PATCH BZ 1088953 69/77] mei: drop pr_fmt macros
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit 80260f2360d34912cd8cb18aca6eda20e9db2ef0
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Tue Mar 11 14:49:24 2014 +0200

    mei: drop pr_fmt macros

    we are using dev_ logging facility so we don't need pr_fmt macros

    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Reviewed-by: Alexander Usyskin <alexander.usyskin@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/main.c b/drivers/misc/mei/main.c
index f862800..7be45db 100644
--- a/drivers/misc/mei/main.c
+++ b/drivers/misc/mei/main.c
@@ -13,9 +13,6 @@
  * more details.
  *
  */
-
-#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
-
 #include <linux/module.h>
 #include <linux/moduleparam.h>
 #include <linux/kernel.h>
diff --git a/drivers/misc/mei/pci-me.c b/drivers/misc/mei/pci-me.c
index 5b76661..45d01d8 100644
--- a/drivers/misc/mei/pci-me.c
+++ b/drivers/misc/mei/pci-me.c
@@ -13,9 +13,6 @@
  * more details.
  *
  */
-
-#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
-
 #include <linux/module.h>
 #include <linux/moduleparam.h>
 #include <linux/kernel.h>
-- 
1.7.1